tests/cases/fourslash/quickInfoJsDocTagsFunctionOverload.ts

```diff
@@ -0,0 +1,62 @@
+/// <reference path='fourslash.ts'/>
+
+// @Filename: quickInfoJsDocTagsFunctionOverload.ts
+////declare function /*1*/foo(): void;
+////
+/////**
+//// * @tag Tag text
+//// */
+////declare function /*2*/foo(x: number): void
```

Comment:
this test is a good start.

1. can you add tests for documentation as well?
2. can you add a test with the sequences [doc/no-tag, doc/tag], [doc/no-tag, doc/no-tag], [no-doc/no-tag, doc/tag], [no-doc/no-tag, doc/no-tag], [no-doc/no-tag, no-doc/tag]? There are other combinations but those are the ones that come to mind.
3. I couldn't get this to work in my editor when I played with it locally, although the tests pass. Probably I just set it up wrong, but you might want to make sure it at leasts works in your editor.